delfin/exporter/prometheus/alert_manager.py

```diff
@@ -51,7 +51,7 @@ def push_prometheus_alert(self, alerts):
 
             self.alerts.append(dict)
             try:
-                response = requests.post('http://' + host + ":" + port +
+                response = requests.post('https://' + host + ":" + port +
```

Comment:
Fix is not sufficient

Reply:
Can you guide me how to solve this issue please.

Reply:
This requires adding Authentication/Authorization support for the services (This issue is not part of SODACODE)